File: GlnApi.Typescript/src/models/ipr.ts
//Global Location Number (GLN) Registry API
//Copyright (C) 2018  University Hospitals Plymouth NHS Trust
//
//You should have received a copy of the GNU Affero General Public License
//along with this program.  If not, see <http://www.gnu.org/licenses/>. 
// 
// See LICENSE in the project root for license information.
export class Ipr implements IIpr {
    Id: number;
    IprName: string;
    IprImageAddress: string;
    Active: boolean;
}

export interface IIpr {
    Id: number;
    IprName: string;
    IprImageAddress: string;
    Active: boolean;
}

File: GlnApi/Models/Ipr.cs
//Global Location Number (GLN) Registry API
//Copyright (C) 2018  University Hospitals Plymouth NHS Trust
//
//You should have received a copy of the GNU Affero General Public License
//along with this program.  If not, see <http://www.gnu.org/licenses/>. 
// 
// See LICENSE in the project root for license information.
namespace GlnApi
{
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;

    [Table("GLNREGISTRY.Ipr")]
    public partial class Ipr
    {
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int Id { get; set; }

        [StringLength(100)]
        public string IprName { get; set; }

        [StringLength(100)]
        public string IprImageAddress { get; set; }

        public bool? Active { get; set; }
    }
}
